### ISSUE TITLE
Aditional Alexa 5k 10k 20k warning lists

### ISSUE THREAD

**Issue by zoni171** [MISP/misp-warninglists#24]
Would it be possible to add warning lists for:

1. top 5k
2. top 10k
3. top 20k

Some organizations would benefit from these to get rid of potential FPs.

**Comment by adulau** [comment]
Indeed, this is a good idea. The problem is that Alexa stopped to generate their public list. We would need to add [Cisco Umbrella list](https://umbrella.cisco.com/blog/blog/2016/12/14/cisco-umbrella-1-million/) and make a script to create the various top lists.

**Comment by adulau** [comment]
Related to #13

**Comment by zoni171** [comment]
This could possibly be an alternative?

s3.amazonaws.com/alexa-static/top-1m.csv.zip

**Comment by ksatlas** [comment]
While configuring Alexa warning list I notice that the list is static. It contains a "snapshot" of what was the case in Feb 2017. Rankings have changed till then. Is this a dynamic list? And as zoni171 and cert-lexsi mentions, is there a way to configure to a specific number of records? 1000, 5K, 20K etc. Different organizations use different thresholds in this.
Thanks,
CERT-EU

**Comment by adulau** [comment]
By the way, the Cisco Umbrella list is now in the warning-list:

https://github.com/MISP/misp-warninglists/tree/master/lists/cisco_top1000

we could create new list with a specific limit per warning-list.
